samples/unit-tests/series-boxplot/boxplot/demo.js

```diff
@@ -190,4 +190,79 @@ QUnit.test(
             'Y axis max should consider the one valid point'
         );
     }
+);
+
+QUnit.test(
+    'Individual dash styles for box, median, stem and whisker lines (#13065)',
+    function (assert) {
+        var chart = Highcharts.chart('container', {
+
+            chart: {
+                type: 'boxplot'
+            },
+
+            series: [{
+                dashStyle: 'DashDot',
+                medianDashStyle: 'ShortDash',
+                whiskerDashStyle: 'dot',
+                data: [
+                    {
+                        low: 760,
+                        q1: 801,
+                        median: 848,
+                        q3: 895,
+                        high: 965,
+                        boxDashStyle: 'dot',
+                        medianDashStyle: 'dot',
+                        stemDashStyle: 'dot',
+                        whiskerDashStyle: 'dot',
+                        whiskerWidth: 1,
+                        medianWidth: 1
+                    }, {
+                        low: 733,
+                        q1: 853,
+                        median: 939,
+                        q3: 980,
+                        high: 1080
+                    }
+                ]
+            }]
+        });
+
+        var series = chart.series[0],
+            firstPoint = series.points[0],
+            secondPoint = series.points[1];
+
+        ['box', 'medianShape', 'stem', 'whiskers'].forEach(function (elem) {
+            assert.strictEqual(
+                firstPoint[elem].attr('stroke-dasharray'),
+                '1,3',
+                'Point\'s ' + elem + 'DashStyle'
```

Comment:
Friendly reminder for this an other tests: make this a full sentence, e.g. `...dashStyle should be applied to a point`.